### PR TITLE
First draft of sig-content's feature matrix

### PR DESCRIPTION
**Pull request by lsemp3d**
The statuses listed are not currently accurate

### CODE REVIEW

File: FEATURE_MATRIX.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+
+**SIG Content**
+
+| Subsystem | Detail | Feature    | Code | Release | Link | Doc | 
+|:---------|:-------------|:--------|:-------|:------------|:-----------:|:-------:|
+|   Frameworks    ||||||| 
+|           | AzToolsFramework | 🟢 Stable | 🟢 Stable | 🟢 Stable |||
+|           | Lua Integration  | 🟢 Stable | 🟢 Stable | 🟢 Stable || [Doc](https://o3de.org/docs/user-guide/scripting/lua/) |
+|           | Prefabs          | 🟢 Stable | 🟢 Stable | 🟢 Stable ||| 
+|           | Qt for Python    | 🟢 Stable | 🟢 Stable | 🟢 Stable |||
```
Comment (AMZN-daimini) [inline FEATURE_MATRIX.md:10]: Should this also include platform support? Some functionality may still not be available on Linux or Mac.

File: FEATURE_MATRIX.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+|           |   Landscape Canvas | 🟢 Stable | 🟢 Stable   | 🟢 Stable ||| 
+|   Gems    ||||||| 
+|           |   Expression Evaluation   | 🟢 Stable | 🟢 Stable | 🟢 Stable  ||| 
+|           |   Script Events           | 🟢 Stable | 🟢 Stable | 🟢 Stable || [Doc](https://o3de.org/docs/user-guide/scripting/script-events/) | 
+|           |   Script Canvas Developer | 🟢 Stable | 🟢 Stable | 🟢 Stable ||| 
+|  Tools    ||||||| 
+|           | Asset Browser              | 🟢 Stable | 🟢 Stable | 🟢 Stable ||| 
+|           | Asset Editor               | 🟢 Stable | 🟢 Stable | 🟢 Stable ||| 
+|           | Asset Processor Reporting  | 🟢 Stable | 🟢 Stable | 🟢 Stable |||
+|           | EmotionFX (Animation)      | 🟢 Stable | 🟢 Stable | 🟢 Stable |||
```
Comment (AMZN-daimini) [inline FEATURE_MATRIX.md:28]: Nit: It's stylized as EMotionFX

**Review by AMZN-daimini — APPROVED** [review]

File: FEATURE_MATRIX.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+|           |   Script Events           | 🟢 Stable | 🟢 Stable | 🟢 Stable || [Doc](https://o3de.org/docs/user-guide/scripting/script-events/) | 
+|  Tools    ||||||| 
+|           | Asset Browser              | 🟢 Stable | 🟢 Stable | 🟢 Stable ||| 
+|           | Asset Editor               | 🟢 Stable | 🟢 Stable | 🟢 Stable ||| 
+|           | Asset Processor Reporting  | 🟢 Stable | 🟢 Stable | 🟢 Stable |||
+|           | EMotionFX (Animation)      | 🟢 Stable | 🟢 Stable | 🟢 Stable ||| 
+|           | Installation/Update Tool   | 🟢 Stable | 🟢 Stable | 🟢 Stable ||| 
+|           | Lua Editor                 | 🟢 Stable | 🟢 Stable | 🟢 Stable |||
+|           | LyShine (2D Render)        | 🟢 Stable | 🟢 Stable | 🟢 Stable |||
+|           | Whitebox                   | 🟢 Stable | 🟢 Stable | 🟢 Stable |||
```
Comment (AMZN-daimini) [inline FEATURE_MATRIX.md:32]: This is missing TrackView I believe?

Comment (lsemp3d) [reply FEATURE_MATRIX.md]: Thanks, updated

File: FEATURE_MATRIX.md (hunk 1 above)
Comment (lsemp3d) [reply FEATURE_MATRIX.md]: I'll review what other sigs do and follow suit

**Comment by Ulrick28** [comment]
Need to add entries for Installer and Project Manager

File: FEATURE_MATRIX.md [hunk 4]
```diff
@@ -15,7 +15,12 @@
 |              | Graph Model      | 🟢 Stable | 🟢 Stable   | 🟢 Stable |||
 |              | Graph Canvas     | 🟢 Stable | 🟢 Stable   | 🟢 Stable ||| 
 |              | Landscape Canvas | 🟢 Stable | 🟢 Stable   | 🟢 Stable ||| 
-|              | Script Canvas    | 🟢 Stable | 🟠 Volatile | 🟢 Stable || [Doc](https://o3de.org/docs/user-guide/scripting/script-canvas/) |
+|              | Script Canvas    | 🟢 Stable | 🟠 Volatile | 🟢 Stable || [Doc]| Installer    ||||||| 
+| Installer    ||||||| 
+|              | Installer  | 🟢 Stable | 🟢 Stable   | 🟢 Stable |||
```
Comment (AMZN-daimini) [inline FEATURE_MATRIX.md:20]: I thought Installer was under release?

Comment (Ulrick28) [reply FEATURE_MATRIX.md:20]: Installer is sig-build, Project Manager is sig-content. Sig-release does not own features, save the actual releases themselves.

Comment (AMZN-daimini) [reply FEATURE_MATRIX.md:20]: Ok will keep that in mind :D

Comment (lsemp3d) [reply FEATURE_MATRIX.md:20]: I'll remove it

**Pull request merged**